docs/datastore/datastore.md

```diff
@@ -27,7 +27,7 @@ K3s supports the following datastore options:
   * [PostgreSQL](https://www.postgresql.org/) (certified against versions 10.7, 11.5, and 14.2)
 
 :::caution Prepared Statement Support
-K3s requires prepared statements support from the DB. This means that connection poolers such as [PgBouncer](https://www.pgbouncer.org/faq.html#how-to-use-prepared-statements-with-transaction-pooling) will not work with K3s.
+K3s requires prepared statements support from the DB. This means that connection poolers such as [PgBouncer](https://www.pgbouncer.org/faq.html#how-to-use-prepared-statements-with-transaction-pooling) may require additional configuration to work with K3s.
```

Comment:
As PgBouncer still doesn't not have a public release that supports prepared statements, this section should not change. 
```suggestion
K3s requires prepared statements support from the DB. This means that connection poolers such as [PgBouncer](https://www.pgbouncer.org/faq.html#how-to-use-prepared-statements-with-transaction-pooling) will not work with K3s. Other connection poolers, such as newer releases of PgCat, [support prepared statements.](https://github.com/postgresml/pgcat/releases/tag/v1.1.0)
```

Reply:
I thought PgBouncer was going to cut a release soon, but that did not happen. Still, I'd like to wait a few more weeks to see if PgBouncer will release v1.21 in time.

Reply:
PgBouncer 1.21.0 was released on 2023-10-16.